scripts/src/modules/search/mobile-filter-expander.js

```diff
@@ -8,15 +9,11 @@ export default function() {
     if(!$searchFilterContainer || !$searchGrid || !$main) {
         return;
     }
+    //need case where validation warning is invoked
 
     //need case where validation warning is invoked
```

Comment:
The comments are duplicated. Should be one line as in https://github.com/nationalarchives/ds-wagtail/blob/39cb00d0798be18967e7f5269e7312ead95b8229/scripts/src/modules/search/mobile-filter-expander.js#L12